docs/user/rstudio/ide/guide/ui/ui-panes.qmd

```diff
@@ -2,54 +2,54 @@
+The guide below provides information based on the default layout/options of each pane. The specific components within each pane and overall arrangement of panes can be modified by opening the **Tools \> Global Options** \> **pane Layout**.
 
 Additional source columns can be added or removed to the left of the 4 primary panes with the **Add Column** or **Remove Column** buttons.
 
-![](images/pane-layout.png){fig-alt="Screenshot of the Tools > Global Options > Pane Layout page inside RStudio."}
+![](images/pane-layout.png){fig-alt="Screenshot of the Tools > Global Options > pane Layout page inside RStudio."}
 
 ## Source
 
-The source pane allows users to view and edit various code-related files, such as .R, .rmd, .qmd, .py, .css, or general text files such as .txt or .md. By default it is the top-left panel and can be launched by opening any editable file in RStudio. Each additional file that is opened will be added as a new tab within the Source Pane.
+The source pane allows users to view and edit various code-related files, such as .R, .rmd, .qmd, .py, .css, or general text files such as .txt or .md. By default it is the top-left panel and can be launched by opening any editable file in RStudio. Each additional file that is opened will be added as a new tab within the Source pane.
 
-Additional source "columns" can be added from the **Global Options** > **Pane Layout** > **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source Pane.
+Additional source "columns" can be added to the left of the four primary panes. To add additional source columns, from the RStudio menu: **Global Options** \> **pane Layout** \> **Add Source Column**, which provides the ability to work on two (or more) source files at a time within the Source pane.
 
-Individual source files can also be opened in their own window, rather than only within the **Source Pane**.  Source Windows allow you to edit files outside the main RStudio window. This is useful for splitting your work among multiple monitors, or devoting more space to your editor.
+Individual source files can also be opened in their own window, rather than only within the **Source pane**. Source windows allow you to edit files outside the main RStudio window. This is useful for splitting your work among multiple monitors, or devoting more space to your editor.
 
 ### Creating Source Windows
 
 There are two ways to open a new source window:
 
 **Pop out an editor**: click the *Show in New Window button* in any source editor tab.
 
-![](images/source-popout.png){fig-alt="A screenshot of the Source Pane toolbar, highlighting the Source Popout button"}
+![](images/source-popout.png){fig-alt="A screenshot of the Source pane toolbar, highlighting the Source Popout button"}
```

Comment:
This seems to look slightly different in the current daily.

![image](https://user-images.githubusercontent.com/9591545/191807675-1dc9e7d0-2b13-4838-96fd-ad85e33f019c.png)

Reply:
Resolved with updated screenshot from 2022.11.0-daily+203